projects/nordic/nrf52840-dk/ses/aws_tests/aws_tests.emProject

```diff
@@ -888,6 +898,7 @@
             <file file_name="../../../../../vendors/nordic/nRF5_SDK_15.2.0/modules/nrfx/mdk/ses_startup_nrf52840.s" />
             <file file_name="../../../../../vendors/nordic/nRF5_SDK_15.2.0/modules/nrfx/mdk/ses_startup_nrf_common.s" />
             <file file_name="../../../../../vendors/nordic/nRF5_SDK_15.2.0/modules/nrfx/mdk/system_nrf52.c" />
+            <file file_name="../../../../../vendors/nordic/nRF5_SDK_15.2.0/modules/nrfx/mdk/nrf52840.h" />
```

Comment:
Same here. If previously things work "fine" without this header, I wonder whether including it will make a big difference.

Reply:
The include file is actually being used and the path is already part of the include paths. It's just that file was not added to project.